distributed/comm/ucx.py

```diff
@@ -359,10 +359,22 @@ def _scrub_ucx_config():
 
     # if any of the high level flags are set, as long as they are not Null/None,
     # we assume we should configure basic TLS settings for UCX
-    if any([dask.config.get("ucx.nvlink"), dask.config.get("ucx.infiniband")]):
-        tls = "tcp,sockcm,cuda_copy"
+    if any(
+        [
+            dask.config.get("ucx.tcp"),
+            dask.config.get("ucx.nvlink"),
+            dask.config.get("ucx.infiniband"),
+        ]
+    ):
```

Comment:
Since we have this condition now, I think we should error out if none of them are specified. This would still mean we leave things to the UCX default, but we haven't really tested that well and we know it doesn't work in certain cases (e.g., DGX-2 where the number of transports is higher than 64).